src/test/scala/cromwell/engine/WorkflowManagerActorSpec.scala

```diff
@@ -30,7 +30,7 @@ class WorkflowManagerActorSpec extends CromwellTestkitSpec("WorkflowManagerActor
 
   "A WorkflowManagerActor" should {
 
-    val TestExecutionTimeout = 5000 milliseconds
+    val TestExecutionTimeout = 5 seconds
```

Comment:
Try `5.seconds.dialated`, for @coreone 

http://doc.akka.io/docs/akka/2.3.12/scala/testing.html#Accounting_for_Slow_Test_Systems